presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -885,7 +885,7 @@ public boolean supportsLimit()
     }
 
     @Override
-    public boolean isLimitGuaranteed()
+    public boolean isLimitGuaranteed(ConnectorSession session)
```

Comment:
typo in cmt msg: ConnectionSession -> ConnectorSession

Reply:
Yeah, I've noticed that but forgot to force push :)